Composite/Data/DataMetaDataFacade.cs

```diff
@@ -198,11 +198,11 @@ public static DataTypeDescriptor GetDataTypeDescriptor(Guid dataTypeId, bool all
 
             foreach (Assembly assembly in AssemblyFacade.GetLoadedAssembliesFromBin())
             {
-                if (!AssemblyFacade.AssemblyPotentiallyUsesType(assembly, typeof(IData)))
-                {
-                    // Ignoring assemblies that aren't referencing Composite.dll
-                    continue;
-                }
+                //if (!AssemblyFacade.AssemblyPotentiallyUsesType(assembly, typeof(IData)))
+                //{
+                //    // Ignoring assemblies that aren't referencing Composite.dll
+                //    continue;
+                //}
 

```

Comment:
This there a reason for this code block to be commented out?

Reply:
Agree, revert this change